proxies/live/apiproxy/targets/apim.xml

```diff
@@ -21,6 +21,12 @@
           <Name>RaiseFault.403Forbidden</Name>
           <Condition>!(accesstoken.id_token-acr StartsWith "AAL3")</Condition>
         </Step>
+        <Step>
+          <!-- Header NHSD-Request-ID must be in the correct format (GUID) -->
+          <Name>RaiseFault.400BadRequest</Name>
+          <Condition>(original-request-details.header.X-Request-ID != null) and (not original-request-details.header.X-Request-ID ~~ "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$")</Condition>
```

Comment:
I think we're missing the step which creates `original-request-details`. PDS has a policy named `AssignMessage.Copy.RequestHeaders` which does this bit.

Reply:
Although the smoke tests didn't fail so maybe I missed something